Drop agent dump-env command (#324)
NethServer/ns8-core: core/agent/README.md

- `<agent-id>/environment`, copy of additional environment variables passed
  to the action steps. The values are stored in the local module

>   filesystem and copied to Redis if the action has completed successfully

Reviewer: ```suggestion
  filesystem and copied to Redis if the action has completed successfully.
```

### set-env

The `set-env` command modifies an environment variable for subsequent
steps. If the step where the command is invoked is successful, the agent
persists the variable value in the local filesystem `./environment` state

> file and the next step picks it up. For example

Reviewer: ```suggestion
file and the next step will pick it up. For example
```

    set-env FULLNAME "First User"

The environment vars are also copied to the Redis DB under the
`{agent_id}/environment` hash key, if the whole action completes
successfully.

### unset-env

The `unset-env` command removes an environment variable for subsequent
steps. If the step where the command is invoked is successful, the change

> is persisted in the local filesystem `./environment` state file and is
> picked up by the next step. For example

Reviewer: ```suggestion
is persisted in the local filesystem `./environment` state file and will be
picked up by the next step. For example
```
